crossbeam-utils/src/atomic/atomic_cell.rs

```diff
@@ -105,7 +106,8 @@ impl<T> AtomicCell<T> {
     /// // operations on them will have to use global locks for synchronization.
     /// assert_eq!(AtomicCell::<[u8; 1000]>::is_lock_free(), false);
     /// ```
-    pub fn is_lock_free() -> bool {
+    #[const_fn("1.46")]
+    pub const fn is_lock_free() -> bool {
```

Comment:
`#[const_fn]` is proc-macro, but it is very simple and [has no proc-macro related dependencies](https://github.com/taiki-e/const_fn/issues/16), so it has little impact on compile time.